metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -216,6 +216,10 @@ class DBTConfig(StatefulIngestionConfigBase):
         False,
         description="Prior to version 0.8.38, dbt tests were represented as datasets. If you ingested dbt tests before, set this flag to True (just needed once) to soft-delete tests that were generated as datasets by previous ingestion.",
     )
+    enable_only_assertions: bool = Field(
+        False,
+        description="When enabled, inserts only assertion results generated from run_results.json file rather than creating the assertion object"
```

Comment:
What about the rest of the entities (models, ephemeral views, etc?) Are those emitted when this is enabled?

Reply:
Unfortunately no, it works with this:
```yaml
    node_type_pattern:
      allow:
        - ^test.*
```
But it's pretty illogical I think